code/modules/organs/internal/organ_internal.dm

```diff
@@ -62,3 +62,21 @@
 		if (prob(50))
 			take_damage(1,silent=prob(15))
 
+// Vorestation addition begin
```

Comment:
All the code in this file would be better suited in a _vr file to prevent conflicts.